src/Neo.VM/Types/InteropInterface.cs

```diff
@@ -53,7 +53,7 @@ public override int GetHashCode()
         public override T GetInterface<T>()
         {
             if (_object is T t) return t;
-            throw new InvalidCastException($"The item can't be casted to type {typeof(T)}");
+            throw new InvalidCastException($"The item {_object} can't be casted to type {typeof(T)}.");
```

Comment:
If _object.ToString fail, it change the exception

Reply:
fixed.